Description: Fixes to Makefiles
 This patch includes several fixes and adjustments to the Makefiles.
Author: Roland Stigge <stigge@antcom.de>

--- latex2html-2008-debian1.orig/docs/Makefile.frames
+++ latex2html-2008-debian1/docs/Makefile.frames
@@ -26,7 +26,7 @@ LATEX = latex
 MAKEINDEX = makeindex
 BIBTEX = bibtex
 
-L2H = latex2html -no_math -html_version 3.2,math -accent_images textrm -toc_depth 6 
+L2H = latex2html -no_math -html_version 4.0,math -accent_images textrm -toc_depth 6 
 #L2H = l2ng  -no_math -html_version 4.0,math -scalable_fonts 
 
 L2Hframes = latex2html -no_math -html_version 4.0,math,frame -accent_images textrm -scalable_fonts 
@@ -147,13 +147,13 @@ LSEC7	  = $(COMMON) -prefix C -dir $(TOP
 manual.ps:	manual.cb manual.dvi
 	$(DVIPS) manual.dvi -o $@
 
-manual.aux manual.cb:
+manual.aux manual.cb: \
 	manual.tex credits.tex licence.tex overview.tex \
 	support.tex userman.tex hypextra.tex features.tex \
 	problems.tex l2hfiles.dat  manhtml.tex l2hman.sty
 	$(LATEX) manual.tex
 
-manual.glo manual.idx:
+manual.glo manual.idx: \
 	manual.tex credits.tex licence.tex overview.tex \
 	support.tex userman.tex hypextra.tex features.tex \
 	problems.tex l2hfiles.dat  manhtml.tex l2hman.sty
@@ -234,7 +234,8 @@ htmlclean:
 clean:
 	rm -f manual.dvi manual.ps manual.glo manual.log manual.aux \
 	    manual.ilg manual.lof manual.lot manual.cb manual.idx \
-	    manual.toc 
+	    manual.toc manual.gls
+	rm -f *.ptr
 
 
 #
--- latex2html-2008-debian1.orig/docs/Makefile
+++ latex2html-2008-debian1/docs/Makefile
@@ -21,8 +21,8 @@ LATEX = latex
 MAKEINDEX = makeindex
 BIBTEX = bibtex
 
-# L2H = latex2html  -no_math -html_version 3.2,math -scalable_fonts 
-L2H = latex2html  -no_math -html_version 3.2,math 
+# L2H = latex2html  -no_math -html_version 3.2,math -scalable_fonts
+L2H = latex2html  -no_math -html_version 4.0,math
 
 
 #
@@ -30,7 +30,7 @@ L2H = latex2html  -no_math -html_version
 # to set path-variables before calling other scripts.
 # For example...
 #
-TEXMF = /usr/local/texmf
+TEXMF = /usr/lib/texmf
 
 LATEXINPUTDIR = $(TEXMF)/tex/latex//
 PKFONTDIR = $(TEXMF)/fonts/public//pk/cx//
@@ -83,50 +83,50 @@ COMMON	  = -external_file $(TOP) -dir $(
 		-biblio $(REFNODE) -no_auto_link -no_footnode \
 		-split 6 -link 6 -up_url $(TOP).html -up_title $(TOPTITLE)
 
-LTOP	  =  -short_index -split 4 -link 5\
+LTOP	  =  -short_index -split 4 -link 5 -no_auto_link\
 		 -t $(TOPTITLE) $(TOP).tex
 
 LSEC1	  = $(COMMON) -prefix O  \
 		-prev_url $(TOP).html -prev_title $(TOPTITLE) \
-		-down_url $(SUP).html -down_title $(S2TITLE)  $(OVW).tex 
+		-down_url $(SUP).html -down_title $(S2TITLE)  $(OVW).tex
 
 LSEC2	  = $(COMMON) -prefix S  -debug \
 		-prev_url $(OVW).html -prev_title $(S1TITLE) \
-		-down_url $(ENV).html -down_title $(S3TITLE)  $(SUP).tex 
+		-down_url $(ENV).html -down_title $(S3TITLE)  $(SUP).tex
 
-#		-down_url $(MAN).html -down_title $(S3TITLE)  $(SUP).tex 
+#		-down_url $(MAN).html -down_title $(S3TITLE)  $(SUP).tex
 
 LSEC3	  = $(COMMON) -prefix E \
 		-prev_url ../$(TOP)/$(SUP).html -prev_title $(S2TITLE) \
-		-down_url ../$(TOP)/$(HYP).html -down_title $(S4TITLE)  $(ENV).tex 
+		-down_url ../$(TOP)/$(HYP).html -down_title $(S4TITLE)  $(ENV).tex
 
-#		-down_url ../$(TOP)/$(HYP).html -down_title $(S4TITLE)  $(MAN).tex 
+#		-down_url ../$(TOP)/$(HYP).html -down_title $(S4TITLE)  $(MAN).tex
 
 LSEC4	  = $(COMMON) -prefix H \
 		-prev_url ../$(TOP)/$(ENV).html -prev_title $(S3TITLE) \
-		-down_url ../$(TOP)/$(MAN).html -down_title $(S5TITLE)  $(HYP).tex 
+		-down_url ../$(TOP)/$(MAN).html -down_title $(S5TITLE)  $(HYP).tex
 
 #		-prev_url ../$(TOP)/$(MAN).html -prev_title $(S3TITLE) \
-#		-down_url ../$(TOP)/$(FEA).html -down_title $(S5TITLE)  $(HYP).tex 
+#		-down_url ../$(TOP)/$(FEA).html -down_title $(S5TITLE)  $(HYP).tex
 
 LSEC5	  = $(COMMON) -prefix M \
 		-prev_url ../$(TOP)/$(HYP).html -prev_title $(S4TITLE) \
-		-down_url ../$(TOP)/$(PRO).html -down_title $(S6TITLE)  $(MAN).tex 
+		-down_url ../$(TOP)/$(PRO).html -down_title $(S6TITLE)  $(MAN).tex
 
-#		-down_url ../$(TOP)/$(PRO).html -down_title $(S6TITLE)  $(FEA).tex 
+#		-down_url ../$(TOP)/$(PRO).html -down_title $(S6TITLE)  $(FEA).tex
 
 LSEC6	  = $(COMMON) -prefix P \
 		-prev_url ../$(TOP)/$(MAN).html -prev_title $(S5TITLE) \
-		-down_url ../$(TOP)/$(REFNODE).html -down_title $(BIBLIO)  $(PRO).tex 
+		-down_url ../$(TOP)/$(REFNODE).html -down_title $(BIBLIO)  $(PRO).tex
 
 #		-prev_url ../$(TOP)/$(FEA).html -prev_title $(S5TITLE) \
 
-#		-down_url ../$(TOP)/$(CHGNODE).html -down_title $(CHGTITLE)  $(PRO).tex 
+#		-down_url ../$(TOP)/$(CHGNODE).html -down_title $(CHGTITLE)  $(PRO).tex
 
 
 LSEC7	  = $(COMMON) -prefix C -dir $(TOP)  -info 0  \
 		-prev_url ../$(TOP)/$(PRO).html -prev_title $(S6TITLE) \
-		-down_url ../$(TOP)/$(GLOSSNODE).html -down_title $(GLOSS)  $(CHA).tex 
+		-down_url ../$(TOP)/$(GLOSSNODE).html -down_title $(GLOSS)  $(CHA).tex
 
 
 #
@@ -202,12 +202,13 @@ images:
 	$(L2H) -short_index -images_only manual.tex
 
 htmlclean:
-	rm -f manual/.??* manual/*TMP* 
+	rm -f manual/.??* manual/*TMP*
 
 clean:
 	rm -f manual.dvi manual.ps manual.glo manual.log manual.aux \
 	    manual.ilg manual.lof manual.lot manual.cb manual.idx \
-	    manual.toc 
+	    manual.toc manual.gls
+	rm -f *.ptr
 
 
 #
@@ -264,7 +265,7 @@ $(TOP)/$(SUP).html:	$(SUP).tex $(TOP)/$(
 $(SUP):
 	$(L2H) $(LSEC2)
 
-S.stamp:	$(SUP).tex $(TOP)/$(INT) 
+S.stamp:	$(SUP).tex $(TOP)/$(INT)
 	$(L2H) $(LSEC2)
 	touch $@
 
@@ -279,7 +280,7 @@ $(TOP)/$(MAN).html:	$(MAN).tex $(TOP)/$(
 $(MAN):
 	$(L2H) $(LSEC5)
 
-M.stamp:	$(MAN).tex $(TOP)/$(INT) 
+M.stamp:	$(MAN).tex $(TOP)/$(INT)
 	$(L2H) $(LSEC5)
 	touch $@
 
